opta/opta_run_check.py

```diff
@@ -0,0 +1,41 @@
+"""
+Usage: This file is used to check if Opta is already running in the current directory.
+This creates a file in opta directory with the name: .opta_running_<underscored-cwd>_<pid>
+"""
+
+
+import glob
+import os
+
+from opta.constants import HOME
+from opta.exceptions import UserErrors
+
+OPTA_RUNNING_LOC_PREFIX = os.path.join(
+    HOME, ".opta", f".opta_running_{os.getcwd().replace(os.path.sep, '_')}_"
+)
+
+
+def check_opta_running_file_exists() -> bool:
+    return glob.glob(OPTA_RUNNING_LOC_PREFIX + "*").__len__() > 0
+
+
+def create_opta_running_file() -> None:
+    with open(f"{OPTA_RUNNING_LOC_PREFIX}_{os.getpid()}", "w") as f:
```

Comment:
why do we need the pid here?
if we call it `.opta.lock` would that make the code and the user troubleshooting (manually delete) easier?

Reply:
Since currently the `opta_release_lock` is in the `finally` block, if opta is run twice, it would not know whether it is releasing the lock on the current failed run or an already running one.